BibBuddy/app/src/main/java/de/bibbuddy/BookFragment.java

```diff
@@ -455,4 +582,4 @@ public void onLongItemClicked(int position, BookItem bookItem, View v) {
     }
   }
 
-}
+}
```

Comment:
Please add a newline at the end of the file.
```suggestion
}

```